packages/repository/src/__tests__/unit/model/model.unit.ts

```diff
@@ -434,4 +436,41 @@ describe('model', () => {
       firstName: 'Test User',
     });
   });
+
+  describe('rejectNavigationalPropertiesInData', () => {
+    class Order extends Entity {
+      static definition = new ModelDefinition('Order')
+        .addProperty('id', {type: 'string', id: true})
+        .addRelation(<BelongsToDefinition>{
+          name: 'customer',
+          type: RelationType.belongsTo,
+          targetsMany: false,
```

Comment:
This is not very ergonomic. In the future, it would be nice to implement `add*` methods for specific relation types. 

Example usage:

```ts
new ModelDefinition('Order')
  .belongsTo('customer', {
    source: Order,
    target: () => User,
    keyFrom: 'customerId', // can we infer this value?
  });
```

Such change is out of scope of this pull request though.

Reply:
Opened https://github.com/strongloop/loopback-next/issues/5384